ui/src/components/tracks/debug_slice_track_details_panel.ts

```diff
@@ -262,4 +298,71 @@ export class DebugSliceTrackDetailsPanel implements TrackEventDetailsPanel {
   isLoading() {
     return this.data === undefined;
   }
+
+  private async loadArgs(argSetId: number) {
+    const queryRes = await this.trace.engine.query(`
+      SELECT
+        args.id AS id,
+        flat_key AS flatKey,
+        key,
+        int_value AS intValue,
+        string_value AS stringValue,
+        real_value AS realValue,
+        value_type AS valueType,
+        display_value AS displayValue
+      FROM args
+      WHERE arg_set_id = ${argSetId}
+    `);
+
+    const it = queryRes.iter({
+      id: NUM,
+      flatKey: STR,
+      key: STR,
+      intValue: LONG_NULL,
+      stringValue: STR_NULL,
+      realValue: NUM_NULL,
+      valueType: STR,
+      displayValue: STR_NULL,
+    });
+
+    const args: Arg[] = [];
+    for (; it.valid(); it.next()) {
+      const value = parseArgValue(it);
+      args.push({
+        id: asArgId(it.id),
+        flatKey: it.flatKey,
+        key: it.key,
+        value,
+        displayValue: it.displayValue ?? 'NULL',
+      });
+    }
+
+    return args;
+  }
+}
+
+function parseArgValue(it: {
```

Comment:
I feel like displayvalue already basically does this. Why do we need this in the UI?